data/mods/TropiCataclysm/monsters/tropical_mammal.json

```
@@ -124,6 +140,66 @@
     "special_attacks": [ [ "EAT_FOOD", 20 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "PET_MOUNTABLE", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE", "SWIMS" ]
   },
+  {
+    "id": "mon_camel",
+    "type": "MONSTER",
+    "name": { "str": "camel" },
+    "description": "A large herbivore introduced from the old world via zoos and private owners.  Ironically, camels were once native to the american continent in distant times.",
```

```suggestion
    "description": "A large herbivore introduced from the old world via zoos and private owners.  Ironically, camels were once native to the American continent in distant times.",
```

Just American continent sounds wird. Maybe specify if its North or South America or just "native to America".